### PR TITLE
fix:  verify-key-status-codes-docs

### LINKED ISSUE #822: More justification for verify key status codes
### Type of documentation issue

Other

### Describe the problem and the proposed solution

For verify key, it's clear that these are key statuses & the http status will be 200, however, given similar names, its easy to get confused. Maybe a note at the bottom saying "These are key statuses, HTTP status will be 200" can be added. Just incase someone gets confused.

image

### PR DESCRIPTION
**Pull request by Pranavtej**
## What does this PR do?

Fixes #822 (issue)

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [ ] Filled out the "How to test" section in this PR
- [ ] Read [Contributing Guide](./CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand areas
- [ ] Ran `pnpm build`
- [ ] Ran `pnpm fmt`
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [x] Updated the Unkey Docs if changes were necessary

## Summary by CodeRabbit

- **New Features**
	- Introduced an optional `note` field in the key verification response, providing additional context about key statuses.
	- Added an optional `enabled` field in the response to indicate whether the key is enabled or disabled.

- **Improvements**
	- Reformatted the response body for improved clarity, particularly regarding the `identity` object.

### CODE REVIEW

**Review by chronark — CHANGES_REQUESTED** [review]
This introduces a new field which is expected to be returned, that's not what we want.

You probably want to update the descriptin in the `openapi({ ... })` object

**Comment by Pranavtej** [comment]
sure , i will update the code

File: apps/api/src/routes/v1_keys_verifyKey.ts [hunk 1]
```diff
@@ -239,6 +239,7 @@ Possible values are:
 - DISABLED: the key is disabled
 - INSUFFICIENT_PERMISSIONS: you do not have the required permissions to perform this action
 - EXPIRED: The key was only valid for a certain time and has expired.
+These are key statuses, HTTP status will be 200
```
Comment (chronark) [inline apps/api/src/routes/v1_keys_verifyKey.ts:242]:
```suggestion
These are validation codes, the HTTP status will be 200.
```

**Review by chronark — CHANGES_REQUESTED** [review]
sign the CLA please

**Comment by Pranavtej** [comment]
@chronark  Signed the CLA

**Review by chronark — APPROVED** [review]

**Pull request merged**